app/src/main/res/layout/activity_main.xml

```diff
@@ -232,78 +201,60 @@
                 android:scaleType="centerCrop" />
         </androidx.cardview.widget.CardView>
 
+        <ImageButton
+            android:id="@+id/flipCameraButton"
+            style="@style/SelfieFlipCameraButton"
+            android:layout_width="50dp"
+            android:layout_height="50dp"
+            android:contentDescription="@string/flip_camera_button_description"
+            android:scaleType="center"
+            android:src="@drawable/ic_flip_camera_anim"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/shutterButton"
+            app:layout_constraintTop_toTopOf="parent" />
+
         <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="0dp"
             android:layout_height="0dp"
             app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toStartOf="@+id/shutterButton"
-            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/shutterButton"
             app:layout_constraintTop_toTopOf="parent">
 
-            <ImageButton
-                android:id="@+id/flipCameraButton"
-                style="@style/SelfieFlipCameraButton"
-                android:layout_width="50dp"
-                android:layout_height="50dp"
-                android:contentDescription="@string/flip_camera_button_description"
-                android:scaleType="center"
-                android:src="@drawable/ic_flip_camera_anim"
-                app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintHorizontal_bias="0.5"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/constraintLayout" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
 
-            <androidx.constraintlayout.widget.ConstraintLayout
-                android:id="@+id/constraintLayout"
-                android:layout_width="135dp"
-                android:layout_height="50dp"
-                android:background="@drawable/layout_camera_mode"
-                app:layout_constraintBottom_toTopOf="@+id/flipCameraButton"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintHorizontal_bias="0.5"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent">
+    <com.google.android.material.chip.ChipGroup
```

Comment:
This layout doesn't let me evenly align the children, guess we'll have to use a plain old linear layout + our own items ( MaterialButtons? )